gslab_scons/provenance.py

```diff
@@ -129,8 +132,13 @@ def write_readme(readme_path, provenance_path):
     '''
     Writes readme to provenance.
     '''
-    with open(readme_path, 'rU') as f:
-        out = '%s' % f.read()
+    try:
+        with open(readme_path, 'rU') as f:
+            out = '%s' % f.read()
+    except IOError:
```

Comment:
Think we could shorten these error messages to 80 characters per line?

Reply:
GSLab standard is 100 cols no?

Reply:
Yes, but pep is 80, and I think it makes things easier to read. Leave it if you want.